cli/commands/caddy.js

```diff
@@ -0,0 +1,84 @@
+const utils = require("../utils")
+const os = require("node:os")
+const { PrismaClient } = require("@prisma/client");
+const prisma = new PrismaClient();
+const validator = require('validator');
+
+module.exports = function ({ program, run }) {
+    const caddy = program.command("caddy")
+    var username = os.userInfo().username
+    caddy
+        .command('list')
+        .description('lists all domains you have configured in caddy')
+        .option('--user', 'allows you to add a domain on behalf of a user (requires sudo)')
```

Comment:
```suggestion
        .option('--user', 'allows you to list a different user's domains (requires sudo)')
```
also - where is the sudo check?

Reply:
fixed + added